### PR TITLE
Add ballerina package files

### PR DESCRIPTION
**Pull request by ViduraRavihansa**
## Purpose
This package contains a record type in the ballerina language, and records can be used to map objects and do operations in a easy manner. This particular record type can be used to map the scim object and access its fields easily.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: scim/scim.bal [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# + addresses - Address of the user
+# + emails - Email Address of the user
+# + ims - IM of the user
+# + meta - Meta infomation
+# + name - User given name and family name
+# + password - Password od the User
```
Comment (LakshanSS) [inline scim/scim.bal:22]:
```suggestion
# + password - Password of the User
```

File: scim/scim.bal [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    string[] ims?;
+    json meta;
+    json name?;
+    string password?;
+    json[] phoneNumbers?;
+    string profileUrl?;
+    string[] schemas;
+    string userName;
+    
+};
```
Comment (LakshanSS) [inline scim/scim.bal:40]:
```suggestion
public type SCIMUser record {
    json[] addresses?;
    string[] emails?;
    string[] ims?;
    json meta;
    json name?;
    string password?;
    json[] phoneNumbers?;
    string profileUrl?;
    string[] schemas;
    string userName;
};
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,14 @@
...
+* [ballerina record types](https://ballerina.io/learn/by-example/records/)
+
+## To use this package in the ballerina code
+Import the package,
+
+```ballerina
+import ballerinax/scim
+```
+## To create your own ballerina package
+[Creating a ballerina package](https://docs.google.com/document/d/17w11v-Jt8N2afKQm8TB6zpDyMwN3Vbub/edit?usp=sharing&ouid=101373359904096888710&rtpof=true&sd=true)
```
Comment (LakshanSS) [inline README.md:14]: Is it necessary to add this part here?

**Comment by NipunaRanasinghe** [comment]
Closing as outdated. @ViduraRavihansa please feel free to reopen after updating the PR, if this is still relevant

**Pull request closed without merging**